src/app/components/carousel/index.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import classnames from 'classnames';
+import ReactSwipe from 'react-swipe';
+
+const Carousel  = React.createClass({
```

Comment:
👍 Ah this is actually not part of the homepage. I'm not sure when this component was made but it must've been before the syntactic sugar update in React. I intend on bringing all components into line eventually. However, components written with this ES6 are not compatible with mixins so I will have the bigger task of removing mixins first!